RestrictEvents/SoftwareUpdate.cpp

```diff
@@ -148,7 +148,16 @@ static int my_sysctl_vmm_present(__unused struct sysctl_oid *oidp, __unused void
 	char procname[64];
 	proc_name(proc_pid(req->p), procname, sizeof(procname));
 	// SYSLOG("supd", "\n\n\n\nsoftwareupdated vmm_present %d - >>> %s <<<<\n\n\n\n", arg2, procname);
-	if (revsbvmmIsSet && (strcmp(procname, "softwareupdated") == 0 || strcmp(procname, "com.apple.Mobile") == 0)) {
+	if (revsbvmmIsSet && (
+		// Always return 1 in recovery/installers
+		(lilu.getRunMode() & LiluAPI::RunningInstallerRecovery) ||
```

Comment:
Also explicitly state `!= 0` for consistency please. Otherwise looks good to me.

Reply:
Thanks for looking it over!